src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -8886,6 +8894,10 @@ internal static class CopyFileRemoteUtils
 
             [string] $createDirectoryPath,
 
+            [Parameter(ParameterSetName=""PSCreateDirectoryOnRemoteSession"", Mandatory=$true)]
```

Comment:
To make the feature experimental, you will need to decide whether to pass in `-Attributes` based on if the experimental feature is turned on, so this cannot be `Mandatory`.